### DISCUSSION TITLE
Can we input custom genomes?

### DISCUSSION THREAD

**Discussion by yabeykoon** [Katsevich-Lab/sceptre#161] [Q&A] [answered]
SCEPTRE works amazing for perturb-seq data aligned to hg38. Now we have a new dataset of macaque cell perturb-seq data aligned to rheMac10. Both human and macaque cells were pooled at sequencing, so the Cellranger output has both human and macaque cells.
1.  Is there a way I can input rheMac10 genome to SCEPTRE for analysis? Maybe there's a way to edit `response_position_data_frame` to reflect rheMac10 genes?
2. Unfortunately when cells from 2 different species are pooled at sequencing, SCEPTRE doesn't even work on the human cells in the pool. I imagine I can preprocess data and input only human cells as preprocesses R objects as in Step 1.2 of the handbook?

**Comment by ekatsevi** [comment, marked as answer]
I believe the only place where the reference genome matters in `sceptre` is in the function [construct_cis_pairs()](https://katsevich-lab.github.io/sceptre/reference/construct_cis_pairs.html). In this function, you may specify an arbitrary reference genome via the argument `response_position_data_frame`. Just structure the data frame you pass to the argument in the same way that `gene_position_data_frame_grch38` is structured.

While I'm not sure in what way `sceptre` doesn't work when you input cells from the two species, I recommend you split apart the data from the two species for your analysis (unless you have a good reason to analyze the two together). To split the data, indeed, one option is to get all your data into R, then split it by species, then import each into `sceptre` via `import_data()`.
